Makefile

```diff
@@ -5,14 +5,17 @@ TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
 DCOS_VERSION := 1.11
 KUBERNETES_VERSION ?= 1.10.4
 KUBERNETES_FRAMEWORK_VERSION ?= 1.1.1-1.10.4
+# PATH_TO_PACKAGE_OPTIONS holds the path to the package options file to be used
+# when installing DC/OS Kubernetes.
+PATH_TO_PACKAGE_OPTIONS ?= "$(PWD)/.deploy/options.json"
 
-# Set PATH to include local dir for locally downloaded binaries.
-export PATH := .:$(PATH)
+# Set PATH (locally) to include local dir for locally downloaded binaries.
+FAKEPATH := "$(PWD):$(PATH)"
 
 # Get the path to relevant binaries.
-DCOS_CMD := $(shell PATH=$(PATH) command -v dcos 2> /dev/null)
-KUBECTL_CMD := $(shell PATH=$(PATH) command -v kubectl 2> /dev/null)
-TERRAFORM_CMD := $(shell command -v terraform 2> /dev/null)
+DCOS_CMD := $(shell PATH=$(FAKEPATH) command -v dcos 2> /dev/null)
+KUBECTL_CMD := $(shell PATH=$(FAKEPATH) command -v kubectl 2> /dev/null)
+TERRAFORM_CMD := $(shell PATH=$(FAKEPATH) command -v terraform 2> /dev/null)
```

Comment:
we do not download `terraform` cli, so needs to check for it in the local folder.

Reply:
@rimusz yes, I understand that, but it also doesn't hurt to place this check here for "consistency". If `terraform` is not found locally, the next directory in the `PATH` will be used.

Reply:
ok, let's keep this way then